harness/determined/common/api/b.py

```diff
@@ -0,0 +1,211 @@
+import enum
```

Comment:
this file is generated. but not really generated here. I'm working on wrapping up the codegen with @rb-determined-ai help but this should do for now and we'll be able to just replace it later.